lib/rubocop/cop/style/unneeded_percent_q.rb

```diff
@@ -14,6 +14,7 @@ def on_dstr(node)
         end
 
         def on_str(node)
+          return unless standalone_string?(node)
```

Comment:
Hmm, that's odd. A string with interpolation is actually a `dstr` and we have a check for it in `on_dstr`. I guess we can do more or less the same check here.